app/src/main/java/com/example/recyclerviewprogram/Users.java

```diff
@@ -0,0 +1,23 @@
+package com.example.recyclerviewprogram;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Users implements Serializable {
+    private static final Users INST = new Users();
+
+    private final List<Item> items = new ArrayList<>();
+
+    private Users(){}
```

Comment:
If we do singleton, we need synchronized singleton, because not work in one thread